catalog/ui/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/geocoder.js

```diff
@@ -17,11 +17,12 @@ var Backbone = require('backbone');
 
 var Cesium = require('cesium');
 var announcement = require('component/announcement');
+var properties = require('properties');
 
 // Note: using a non-secure resource will fail when running DDF with TLS.
 var geocoderOnlineEndpoint = 'https://nominatim.openstreetmap.org/search?format=json&q=';
 var geocoderOfflineEndpoint = '/services/REST/v1/Locations?jsonp=loadJsonp&key=0&query=';
-var onlineGazetteer = true;
+var onlineGazetteer = properties.onlineGazetteer;
```

Comment:
❓ Do we want to possibly put in a default in case that property is missing?
E.g, `var onlineGazetteer = properties.onlineGazetteer !== undefined ? properties.onlineGazetteer : true;`

Reply:
The way the property is used right now it will default to the offline one in this case. I think that is probably ok unless you feel it should default to use the online one.